external-builds/pytorch/smoke-tests/README.md

```diff
@@ -0,0 +1,28 @@
+# PyTorch Smoke Tests
+
+This repository contains a set of basic smoke tests for verifying fundamental PyTorch tensor operations. These tests ensure that basic matrix operations function correctly, but they are **not exhaustive PyTorch tests**.
+
+## Disclaimer
+
+⚠️ **These tests are not full PyTorch tests.** They are designed as quick checks for basic tensor operations and do not replace comprehensive testing of PyTorch functionalities.
+
+## Test Overview
+
+The following operations are covered in these smoke tests:
+
+- **Matrix Multiplication (`torch.mm`)**
+- **Batch Matrix Multiplication (`torch.bmm`)**
+- **Matrix Multiplication using `@` operator**
+- **Element-wise Multiplication (`*`)**
+- **Matrix Transposition (`torch.t`)**
+- **Dot Product (`torch.dot`)**
+- **Matrix-Vector Multiplication (`torch.mv`)**
+- **General Matrix Multiplication (`torch.matmul`)**
+
+## Running the Tests
+
+To run the tests, ensure you have PyTorch installed:
+
+```bash
+PYTORCH_TEST_WITH_ROCM=1 python3 test/run_test.py --verbose
+```
```

Comment:
This comment does not look accurate. We don't have a `PYTORCH_TEST_WITH_ROCM` environment variable in this repository yet and the single test file is named differently.